CMakeLists.txt

```
@@ -36,7 +38,8 @@ target_compile_definitions(
 set_target_properties(
     tinyxml2
     PROPERTIES
-    DEFINE_SYMBOL "TINYXML2_EXPORT"
+    DEFINE_SYMBOL "TINYXML2_EXPORT" # only used when tinyxml2_SHARED_LIBS/BUILD_SHARED_LIBS is ON
+

```

So now there's an extra newline? 
```suggestion
    DEFINE_SYMBOL "TINYXML2_EXPORT"
```
Let's keep this change functional, please. We can discuss adding proper documentation for this later.